app/views/static/stgall/21198-zz00161mpp-msdesc-updated.html.erb

```diff
@@ -17,7 +17,7 @@
         </div>
         <dl class="document-metadata document__list-metadata-group"></dl>
         <div class="stgall-sub-links">
-            <button class="toggleLinkWorkItem" data-target="work-item-1-content" onclick="toggleSection(this,1)">
+            <button class="toggleLinkWorkItem active" data-target="work-item-1-content" onclick="toggleSection(this,1)">
```

Comment:
@pghorpade added an example of when the `active` class is added with the js logic